### PR TITLE
Gpu support

### PR DESCRIPTION
**Pull request by nrios14**
Added GPU Support to the Billing Overview and to the Build Environment Overview pages.

### CODE REVIEW

File: user/reference/overview.md [hunk 1]
```diff
@@ -262,6 +262,27 @@ Each tier of instance size delivers more vCPU and RAM resources available at you
 | 2x-large  | 16          | ~64         | requires credits to use; may be limited to certain Plans|
 
 
+## GPU VM Instance Size 
+
+You can choose to run your GPU builds on larger instances size. All you need to do is type the appropriate instance size to your `.travis.yml` using the following tags and one of the available values:
```
Comment (michal-at-travisci) [inline user/reference/overview.md:267]: Can I suggest:


You can choose to run your GPU builds on various GPU "sizes".

**Comment by nrios14** [comment]
Updated the changes suggested by @michal-at-travisci

**Review by stan-travis — APPROVED** [review]
Looks good to me

**Review by stan-travis — APPROVED** [review]
Looks good to me

**Review by stan-travis — APPROVED** [review]

**Review by michal-at-travisci — APPROVED** [review]

**Review by hhjmmmmmmmm — APPROVED** [review]

**Review by hhjmmmmmmmm — APPROVED** [review]

**Pull request merged**